src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -21,29 +25,69 @@ public class DeleteCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the person identified by the NusId used in the displayed person list.\n"
             + "Parameters: NusId (8 digits long, starting with an 'E'). \n"
-            + "Example: " + COMMAND_WORD + " E0123456";
+            + "Example: " + COMMAND_WORD + " id/E0123456\n\n"
+            + "OR Deletes all the people identified by the group used in the displayed person list.\n"
+            + "Parameters: Group (Already existing in the address book). \n"
+            + "Example: " + COMMAND_WORD + " g/CS2103-T15";
```

Comment:
Might want to consider listing down all the functionalities of the command, then provide the usage (examples) together afterwards.